Meta/Lagom/CMakeLists.txt

```diff
@@ -347,18 +351,20 @@ add_serenity_subdirectory(Userland/Libraries/LibFileSystem)
 add_serenity_subdirectory(Userland/Libraries/LibIDL)
 
 # Manually install AK headers
-install(
-    DIRECTORY "${SERENITY_PROJECT_ROOT}/AK"
-    COMPONENT Lagom_Development
-    DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}
-    FILES_MATCHING PATTERN "*.h"
-)
-install(FILES
-    ${Lagom_BINARY_DIR}/AK/Debug.h
-    ${Lagom_BINARY_DIR}/AK/Backtrace.h
-    COMPONENT Lagom_Development
-    DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/AK"
-)
+if (ENABLE_INSTALL_HEADERS)
```

Comment:
This should be moved to AK CMake lists, for sure.

Reply:
Can do, although it may be better-suited for a general CMake cleanup changeset rather than here.

Reply:
For sure, worth a sep PR.